packages/salesforcedx-vscode-core/src/commands/commands.ts

```diff
@@ -276,7 +276,7 @@ export class SelectPrioritizedDirPath extends SelectDirPath {
 export class SelectStrictDirPath extends SelectDirPath {
   public globDirs(srcPath: string, priorityKeyword?: string): string[] {
     const globPattern = priorityKeyword
-      ? path.join(srcPath, '**/', priorityKeyword + '/**/')
+      ? path.join(srcPath, '**/', priorityKeyword + '/')
```

Comment:
Removing the "**" will make the glob return only paths ending with the keyword appear.